### PR TITLE
[Feat]: added history page, remove, clear All

### PR DESCRIPTION
**Pull request by anjali1102**

### CODE REVIEW

File: src/component/Cards/VideoCards/VideoCards.js [hunk 1]
```diff
@@ -1,17 +1,20 @@
...
 
 const VideoCards = ({ _id, title, creator, profile }) => {
   const {
     user: { token },
   } = useSelector((store) => store.auth);
 
   const Dispatch = useDispatch();
+  const Location = useLocation();
 
+  console.log("Location", Location);
```
Comment (dev-enforced) [inline src/component/Cards/VideoCards/VideoCards.js:17]: Can remove this debugger later on

File: src/store/historySlice.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+  "history/getHistory",
+  async (data, thunkAPI) => {
+    const { token } = data;
+    try {
+      const response = await axios.get("/api/user/history", {
+        headers: {
+          authorization: token,
+        },
+      });
+      console.log("gethistory fulfilled", response.data);
```
Comment (dev-enforced) [inline src/store/historySlice.js:79]: Can remove the debugger

**Review by dev-enforced — APPROVED** [review]
The live preview is working as mentioned (https://deploy-preview-5--tangerine-moxie-0c0654.netlify.app/history)
Can remove the debuggers or console.logs related to responses in the thunks added to the slice file. Do check on the deploys if they are appearing to be wrong through the logs
Good work @anjali1102

**Review by MaruthiPothuganti — APPROVED** [review]
Features specified works perfectly with clean code. PR looks good to merge.

**Pull request merged**